.github/workflows/CI.yaml

```diff
@@ -25,12 +27,15 @@ jobs:
       fail-fast: false
       matrix:
         os: [macOS-latest, ubuntu-latest, windows-latest]
-        python-version-min: [3.6, 3.7]
         python-version: [3.7, 3.8, 3.9]
-        
+        env_file: [env, ]
```

Comment:
Here we add this extra variable to pick up the right environment file: `env` for the normal `test_env.yaml` and `min` for `test_min.yaml`.

In this case all the normal jobs in the matrix (which run all 3 OS versions + all the python versions) will use `test_env.yaml`.